tests/util/test_config.py

```diff
@@ -14,6 +14,10 @@
 # class scope.  (The tests below are ordered with this in mind.)
 
 class TestConfig:
+    # Nominally, this tests should pass even if SEECHANGE_CONFIG is not None.  However, because there
+    # are autouse fixtures in ../conftest.py, one of the side effects is that the config default
+    # gets set.
+    @pytest.mark.skipif( os.getenv("SEECHANGE_CONFIG") is not None, reason="Clear SEECHANGE_CONFIG to test this" )
```

Comment:
Is this actually going to get run in the test suite?

Reply:
Not as it exists right now, because of the auto-run frameworks in conftest.py